notebooks/05_operationalize/dbapi_conda.yml

```diff
@@ -0,0 +1,17 @@
+

```

Comment:
Do we need this as most of the dependencies are found in the conda yaml of the repo.

Reply:
I'm fine with dropping it. The original intent was just to make a minimal dependencies file for just running this notebook, with 0 expectation that other things within the recommenders repo would be run.

Reply:
removed in favor using CLI in other PR